src/main/java/com/intuit/graphql/orchestrator/utils/XtextUtils.java

```diff
@@ -32,6 +32,12 @@
 
   public static final String XTEXT_TYPE_FORMAT = "[name:%s, type:%s, description:%s]";
 
+  public static final String FEDERATION_KEY_DIRECTIVE = "key";
```

Comment:
TODO I think we should move constants to its own class... FederationConstants.java